### PR TITLE
Removes broadcast and topic type queries

### PR DESCRIPTION
**Pull request by aaronschachter**
Cleanup from #39, adds an `AutoReplySignupTopic` to better differentiate auto reply topics that do/don't create signups. Refs https://github.com/DoSomething/gambit-conversations/pull/461

### CODE REVIEW

File: src/schema/gambitContent.js [hunk 1]
```diff
@@ -29,11 +29,17 @@ const typeDefs = gql`
...
-    # The campaign to create signup for if conversation changes to this topic (optional).
-    campaignId: Int
+    # The auto reply text.
+    autoReply: String!
+  }
+
+  # Topic for creating a signup and sending a single auto-reply message.
+  type AutoReplySignupTopic implements Topic {
+    ${entryFields}
+    campaignId: Int!
```
Comment (aaronschachter) [inline src/schema/gambitContent.js]: This is what's causing topic 6r3LQmQ6ATPXWI9K0GBcWw to freak out over https://github.com/DoSomething/gambit-conversations/pull/461#issuecomment-455041153 -- it's referencing campaign 9002 which exists in Rogue prod but not Rogue QA (the environment I'm testing against).

File: src/repositories/rogue.js [hunk 2]
```diff
@@ -18,20 +18,16 @@ const ROGUE_URL = config('services.rogue.url');
...
-    );
-
-    const json = await response.json();
-
-    return transformItem(json);
-  } catch (exception) {
-    const error = exception.message;
-    logger.warn('Unable to load campaign.', { id, error, context });
-  }
-  return null;
```
Comment (aaronschachter) [inline src/repositories/rogue.js]: Catching this error was causing the app to timeout when requesting a topic that had a `campaignId` set via Gambini, but no corresponding Rogue campaign. For example, this `AutoReplySignupTopic` 6r3LQmQ6ATPXWI9K0GBcWw has a `campaignId` of 9002 which exists in Rogue Prod but not in Rogue QA (on the environment I was testing on). This is definitely a little bit of a pickle we're in for being able to test conversations, where we don't have a QA instance of Gambit Content.

Comment (aaronschachter) [reply src/repositories/rogue.js]: I'd imagine it makes sense to remove all of the try/catch statements I've added that wrap fetching data (e.g. Gambit [Content](https://github.com/DoSomething/graphql/blob/ee095f660a6c5e93894752bcad1a4f665ea12e90/src/repositories/gambitContent.js#L192) + [Conversations](https://github.com/DoSomething/graphql/blob/ee095f660a6c5e93894752bcad1a4f665ea12e90/src/repositories/gambitConversations.js#L68)) - what do you think @DFurnes ?

Comment (DFurnes) [reply src/repositories/rogue.js]: Huh, that's funny. I'd have thought catching & then returning `null` would work fine here.

**Pull request merged**